.github/workflows/ci.yml

```diff
@@ -35,11 +36,10 @@ jobs:
           if [[ "${{ matrix.mgversion }}" != 1* ]]; then
             COMMUNITY_SUFFIX=""
           fi
-          curl -L https://download.memgraph.com/memgraph/v${{ matrix.mgversion }}/ubuntu-18.04/memgraph_${{ matrix.mgversion }}${COMMUNITY_SUFFIX}-1_amd64.deb > ~/memgraph/memgraph.deb
-      - name: Install system dependencies (Ubuntu 18.04)
+          curl -L https://download.memgraph.com/memgraph/v${{ matrix.mgversion }}/${{ matrix.platform }}/memgraph_${{ matrix.mgversion }}${COMMUNITY_SUFFIX}-1_amd64.deb > ~/memgraph/memgraph.deb
+      - name: Install system dependencies
         run: |
-          sudo apt install -y libpython3.7 python3-pip python3-setuptools
-          python3 -m pip install --upgrade
```

Comment:
My preference is that we should use GitHub actions as much as possible since they make workflows less bug-prone and easier to read, In this case, I would use https://github.com/actions/setup-python to setup python. What do you think about this?

Reply:
I agree that there might be a hidden way how they set up the python environment and I am personally not too familiar with that. So I agree you can ignore this comment. I would like to see these actions used elsewhere(in our repos) to see if they cause some issues. Since your solution works perfectly in our case there is no need to change it for now.